Source/RapyutaSimulationPlugins/Private/Core/RRStaticMeshComponent.cpp

```diff
@@ -34,9 +34,6 @@ void URRStaticMeshComponent::BeginPlay()
     ActorCommon = URRActorCommon::GetActorCommon(SceneInstanceId);
     // [Initialize()] should be invoked right after the mesh comp is created!
     // Please refer to [URRUObjectUtils::CreateMeshComponent()]
-
-    // It is considered that a mesh component should only be physically activated if its Owning Actor is Physically Enabled,
-    // thus not auto activating physics here! Rather, it is left to the Owning actor to do it!
```

Comment:
Same as above : 
Are these comments obsolete for this function? 
Otherwise they may be useful to avoid future bad usage?

Reply:
those are mvoed to header to be accessible from doxygen doc